lib/services/local/access_list.go

```diff
@@ -393,6 +393,22 @@ func (a *AccessListService) GetSuggestedAccessLists(ctx context.Context, accessR
 	return nil, trace.NotImplemented("GetSuggestedAccessLists should not be called")
 }
 
+// GetSuggestedAccessListsForResources returns a list of access lists that are suggested for a set of resources.
+func (a *AccessListService) GetSuggestedAccessListsForResources(ctx context.Context, resourceIDs []string) (*accesslistv1.GetSuggestedAccessListsResponse, error) {
+	// Get all access lists
+	_, err := a.GetAccessLists(ctx)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
```

Comment:
This seems to be NOP and Is to be honest the whole function seems  unnecessary since we already have suggestion login implemented in https://github.com/gravitational/teleport.e/blob/master/lib/accessrequest/suggestion.go#L243